tests/layer_tests/tensorflow_tests/test_tf_Rint.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import tensorflow as tf
+from common.tf_layer_test_class import CommonTFLayerTest
+
+class TestRint(CommonTFLayerTest):
+    def create_tf_rint_net(self, input_shape, input_type):
+        tf.compat.v1.reset_default_graph()
+        with tf.compat.v1.Session() as sess:
+            input = tf.compat.v1.placeholder(input_type, input_shape, 'input')
+            tf.raw_ops.Rint(x=input)
+            tf.compat.v1.global_variables_initializer()
+            tf_net = sess.graph_def
+
+        ref_net = None
+
+        return tf_net, ref_net
+
+    @pytest.mark.parametrize("input_shape", [[6], [2, 5, 3], [10, 5, 1, 5]])
+    @pytest.mark.parametrize("input_type", [tf.half, tf.float32, tf.float64])
```

Comment:
please exclude tf.half because TF complains:

```
No OpKernel was registered to support Op 'Rint' used by {{node Rint}} with these attrs: [T=DT_HALF]
```

I think it does not make much sense to consider this case.